.github/workflows/main.yml

```
@@ -14,7 +14,7 @@ jobs:
     strategy:
       matrix:
         image: [ 'windows-latest', 'ubuntu-latest', 'macos-latest' ]
```

Let's prove that platform detection works for Ubuntu on ARM processors
```suggestion
        image: [ 'windows-latest', 'ubuntu-latest', 'ubuntu-24.04-arm', 'macos-latest' ]
```
* #31

[Standard GitHub-hosted runners for public repositories](https://docs.github.com/en/actions/using-github-hosted-runners/using-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories) ubuntu-24.04-arm

https://github.com/ninja-build/ninja/releases/download/v1.11.1/ninja-linux.zip is not ARM compatible.

    https://github.com/ninja-build/ninja/releases -->
        https://github.com/ninja-build/ninja/releases/download/v1.12.1/ninja-linux-aarch64.zip in v1.12.0 and later.

---

Oh nice, these weren't available when I made the PR. Thanks!